installation/docker/dockercompose.yml

```diff
@@ -0,0 +1,136 @@
+version: '3'
```

Comment:
we might need to think of a way to make sure all the duplications in all the installers are aligned.
For example, a change here in the dockercompose.yaml file will require the same change in 3-4 more files, right?

Reply:
I'm not 100% sure that they will always be aligned or not. If we publish a tarball of the docker compose installer as part of the release process then it could be possible for the AWS/Azure/GCP installers to download it from Github as part of the install process.